src/services/CertificateGenerationService.ts

```diff
@@ -1381,6 +1382,23 @@ class CertificateGenerationService {
       )
     );
   }
+
+  /**
+   * Sorts required required standards if present firstly by sectionNumber and then by rsNumber and then returns it
+   * @param requiredStandards - the requiredStandards collection to sort
+   * @returns - the sorted requiredStandards collection
+   */
+  private sortRequiredStandards = (requiredStandards: IRequiredStandard[] | undefined) : IRequiredStandard[] | undefined => {
+    if(!requiredStandards)
+      return;
+
+    const collator = new Intl.Collator('en', { numeric: true, sensitivity: 'base' });
```

Comment:
" " as per linting and whitespace needed before 'b' in the sort as per linting recommendation